docs/view.md

```diff
@@ -282,6 +282,18 @@ For example, if a touchable view has a height of 20 the touchable height can be
 
 ---
 
+### `id`
+
+Used to locate this view from native classes. Has precedence over `nativeID`
+
+> This disables the 'layout-only view removal' optimization for this view!
```

Comment:
```suggestion
> This disables the 'layout-only view removal' optimization for this view!
```

Can you explain this part a bit broader or provide a reference (link) for the optimization you are referring to?

Reply:
I'm not 100% sure what this is TBH, but this is the exact same wording used in the `nativeID` section

https://github.com/facebook/react-native-website/blob/main/docs/view.md?plain=1#L309

Reply:
Oh, I see so you have just copied it over from the other prop. Let's leave this as-is for now, and I will ask around if someone can clarify this not a bit (or maybe even it's no longer valid, will see).